bliss/simulator/simulated_dataset.py

```diff
@@ -148,12 +140,21 @@ def __getitem__(self, idx):
         return self.data[idx]
 
     def train_dataloader(self):
+        assert self.data, "No cached data loaded; run `generate.py` first"
```

Comment:
Nit: `s/No cached data \(.*\)/No cached train data \1/'`

Reply:
Fixed, changed to "No cached train data"